articles/File_download.md

```diff
@@ -0,0 +1,60 @@
+# Download file from a website and upload it to Dropbox 
+
+## My very first function in powershell ~
+
+```powershell
+function Get-UrlInfo {
+        [Cmdletbinding()]
+        param(
+                [Parameter(Mandatory=$true)]
+                [String]$fileName
```

Comment:
Is there is a real need to actually rely on a `fileName` - maybe we could just use the url string as an input. What do you think?

Reply:
No specific reason, I was just reading the content from a file and then trying to parse it. 
Using URL it should look something like this, right? (btw, I will add the url as input to my function later)
```powershell
PS /root> $inputString = "https://university.business-science.io/courses/541207/lectures/15050206?wvideo=9gj4s6uckw"
PS /root> $videoId = [System.Uri]$inputString
PS /root> $videoUrl = $videoId.Query.Split("=")[1]
PS /root> $link = "https://fast.wistia.net/embed/iframe/" + $videoUrl
PS /root> $link
https://fast.wistia.net/embed/iframe/9gj4s6uckw
PS /root> youtube-dl $link
```

Reply:
Cool. 👍 

BTW, the `input to a function` is called a `parameter` :)

Reply:
```powershell
function Save-Video {
        [Cmdletbinding()]
        param(
                [Parameter(Mandatory=$true)]
                [String]$urlString
        )
        $videoId = ([System.Uri]$urlString).Query.Split("=")[1]
        $videoUrl = "https://fast.wistia.net/embed/iframe/" + $videoId
        youtube-dl $videoUrl
        $outputFile = Get-ChildItem *.bin -Name
        $fileString = $outputFile.Split("-")[0]+"-$videoId" + ".mp4"
        $renameFile = mv $outputFile $fileString
        rclone copy *.mp4 remote:/Videos/ -v
}
```

Reply:
Cool 👍 

Let's get this rolling forward and then add changes later.